### DISCUSSION TITLE
图元覆盖在实体上方时，实体的鼠标事件怎么触发

### DISCUSSION THREAD

**Discussion by cxk-china** [zouyaoji/vue-cesium#271] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/73694384/188459619-10604ff1-b215-479d-9340-91832bb7c654.png)
海水层在点层上方，点的mouseover事件触发不到

**Comment by zouyaoji** [comment, marked as answer]
可以尝试把实体点深度检测距离改为无限大，不行的话，实体也考虑用成图元？

再没招就只有参考 vue-cesium的绘制组件那样魔改一下了。

**Reply by cxk-china** [reply]
深度检测距离disableDepthTestDistance="50000"可以解决，谢谢大佬！
